config-templates/redhat/ocs-remote/4.7/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "4.7",
+  "enabled": "true",
+  "description": "OCS template for openshift 4.7 (Remote storage)",
+  "tags": [
+    {
+      "name": "provider",
+      "value": "redhat"
+    },
+    {
+      "name": "provider-logo",
+      "value": "logo/ocs-remote.svg"
+    },
+    {
+      "name": "kube-version",
+      "value": "1.18"
+    },
+    {
+      "name": "os-version",
+      "value": "RHEL7"
+    }
+  ]
+}
```

Comment:
Metadata list is incomplete, add other fields `offering-name`, `provider-url`, `support-url`,   `volume-type` 
and other pls refer to https://github.com/IBM/ibm-satellite-storage/blob/develop/config-templates/aws/aws-efs-csi-driver/1.2.2/metadata.json